common/app/common/metrics.scala

```diff
@@ -256,5 +261,5 @@ class CloudWatchMetricsLifecycle(
 
 object bytesAsMb {
   // divide by 1048576 to convert bytes to MB
-  def apply(bytes: Long): Long = bytes / 1048576
+  def apply(bytes: Long): Double = (bytes / 1048576).toDouble
```

Comment:
Refactored this method to avoid adding `toDouble` everywhere it is called. Starts from here: https://github.com/guardian/frontend/blob/18790125bce889e814a5e0f37f76c5c2858967b9/common/app/common/metrics.scala#L44